uk.co.compucorp.civicrm.hrleaveandabsences/scss/mixins/_utility.scss

```diff
@@ -0,0 +1,9 @@
+@mixin uniform-padding($width) {
+  padding-left: $width/2;
+  padding-right: $width/2;
+
+  > * {
```

Comment:
That could cause problems. I would specify exact types of elements to have that padding. Are they `col-*`?

Reply:
As discussed, the idea is to apply this style for any child item, hence keeping it.